com.unity.multiplayer.mlapi/Editor/CodeGen/NetworkBehaviourILPP.cs

```diff
@@ -458,13 +457,13 @@ private void InjectWriteAndCallBlocks(MethodDefinition methodDefinition, CustomA
             var instructions = new List<Instruction>();
             var processor = methodDefinition.Body.GetILProcessor();
             var isServerRpc = rpcAttribute.AttributeType.FullName == CodeGenHelpers.ServerRpcAttribute_FullName;
-            var isReliableRpc = true;
+            var rpcDelivery = RpcDelivery.Reliable;
```

Comment:
Do you lose the need for the cast if you instead do:

```suggestion
            RpcDelivery rpcDelivery = RpcDelivery.Reliable;
```

Reply:
nope! `var` is just a shorthand instead of typing full type name. in this case a preferred way since we exactly know what the type is next to variable declaration.